oonib/oonibackend.py

```diff
@@ -64,9 +64,9 @@ def getEndpoint(endpoint_config):
 
 def createService(endpoint, role, endpoint_config):
     if role == 'bouncer':
-        factory = ooniBouncer
+        factory = OONIBouncer()
```

Comment:
Earlier `ooniBouncer` was only instantiated if `config.main.bouncer_file`, now it is instantiated here if `role == 'bouncer'`. This comment is just to check that this change has no unexpected implications.